pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -910,3 +910,13 @@ func (l *buildLoader) AddToGraph(g osgraph.Graph) error {
 
 	return nil
 }
+
+// tolerateNotFoundErrors is tolerant of not found errors in case builds are disabled server
+// side (Atomic).
+// TODO check swagger for feature enablement
```

Comment:
move the TODO to where we add bcLoader and buildLoader to the list of loader functions, and include a comment about removing `tolerateNotFoundErrors` when we do